utils/refl_table_utils.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+from dxtbx.model import ExperimentList
+from dxtbx.model.experiment_list import ExperimentListFactory
+from dials.array_family.flex import reflection_table
+from dials.array_family import flex
+from IPython import embed
+
+expt_file = "dials_temp_files/mega_ultra_refined.expt"
```

Comment:
Library functions should not have hardcoded file names.

Reply:
If you want to put stuff like this in a library, you should put it at the bottom after an `if __name__=="__main__":` clause.

Reply:
Fixed in [https://github.com/Hekstra-Lab/laue-index-assigner/commit/aa952838d50db7ee2aef4933ca47cba19037c686](url)